integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java

```diff
@@ -98,6 +103,66 @@
   }
 
   /**
+   * Changes the mode of an Alluxio file.
+   *
+   * @param path the path of the file
+   * @param mode the mode to change to
+   * @return 0 on success, a negative value on error
+   */
+  @Override
+  public int chmod(String path, @mode_t long mode) {
+    final AlluxioURI uri = mPathResolverCache.getUnchecked(path);
+
+    SetAttributeOptions options = SetAttributeOptions.defaults().setMode(new Mode((short) mode));
+    try {
+      mFileSystem.setAttribute(uri, options);
+    } catch (IOException | AlluxioException e) {
+      LOG.error("Exception on {}", path, e);
```

Comment:
include the operation which failed (`setAttribute`) and the `mode` that it failed with